### ISSUE TITLE
As a moderator, I want to create and edit people in wagtail so that I can manage them just like pages.

### ISSUE THREAD

**Issue by thatbudakguy** [Princeton-CDH/cdh-web#186]
# testing notes
## as a moderator
to test this, log into the new admin site (`/cms`) and check to see if you have a menu on the left side called "People". 

1. Make a new person by going to the "People" menu, choosing "People", and clicking "new person" at the top right. Feel free to also give the person a picture, and assign them one (or multiple) CDH positions at the bottom.
2. Create a profile for your new person by going to the "People" menu, choosing "Profiles", and clicking "add profile" at the top right. Associate the profile with the person you made in step 1, but feel free to upload a _different_ image, and also give the profile a title that isn't the same as the person's name (add a middle initial, for example).
3. Visit the new profile by clicking "preview" at the bottom (by "publish"). Check that the profile is visually similar to current CDH profiles (e.g. your own profile on the current site). The picture displayed should be the picture you uploaded to the profile (not the person) and the name shown at the top should be the profile's title (not the person's name).

you can also manage the list of available CDH position titles. you should see it as "Titles" underneath "People" on the left.

1. Create a new CDH position title by going to the "People" menu, choosing "Titles", and clicking "add title" at the top right. Don't worry about "sort order" for now since it's only relevant for #267; anything is fine.
2. Give the person that you created in step 1 above (as an editor) this new title.
3. Repeat step 3 above and check that the person's profile shows their new job title.

# dev notes
For `Person`, how we can make it orderable and what type of chooser block will we need (maybe extend)? to add it to other pages? We want to preserve the Django user model, along with some functionality to populate it from LDAP, but no longer make Person inherit it.

For person index pages, see #267.
## models
- [x] `Title` (migrations: change user to person, same as membership)
- [x] `Position` (migrations: change user to person, same as membership)
- [x] `Person` (no longer inherits `User` - see board for fields; optional 1:1 with user)
- [x] `Membership` and `Project` (migration: change user to person)
- [x] `PersonResource` (will become wagtail `PersonRelatedLink` via #181)
- [x] new `ProfilePage` (based on wagtail `Page` - migrate profile-specific fields and link to Person; see board for fields; required 1:1 with person)
- [x] update `init_profile_from_ldap` to create new person 1-1 and populate fields
## admin
- [x] configure `Person` in wagtail admin
- [x] configure `Title` in wagtail admin
- [x] make `Position` inline-editable on `Person` in Wagtail
- [x] configure `ProfilePage` in wagtail admin

**Comment by cvansant** [comment]
It would not allow me to create a title without a person (i.e. "positions") to go with it.

It would not allow me to create a person without a preexisting "user."

I don't know how to create a "user."

**Comment by thatbudakguy** [comment]
@cvansant user should be optional now, sorry about that! give it another go.

**Comment by cvansant** [comment]
@thatbudakguy Made it all the way to step 4. When I try to preview the profile, I get the chicken and 500 error. I did give the person a middle initial and a new photo.

**Comment by rlskoeser** [comment]
@thatbudakguy I experimented with this in my local dev and wasn't able to duplicate the 500 error. If we can duplicate it again in qa we should make sure to look at the django logs right away so we can investigate.

**Comment by thatbudakguy** [comment]
- [ ] test previewing profile as superuser
- [ ] test previewing profile as non-superuser

**Comment by manawinters** [comment]
I was able to add a student and her image to People. I was then able to create a profile with a slightly different name and different image. And I was able to preview the new profile (with the name and image I uploaded in Profiles) and confirm that the image appears as it should. 

As a user, I noticed that a website (Rockefeller College) appears as a link next to her profile image. I did not add that url anywhere in her profile (I did use an image of Rockefeller College for the profile, however).

As an editor, I was not able to add a new title ("the title could not be created due to errors") without selecting a position, and when I selected a position (student's name), I received the chicken.
